### PR TITLE
feat: cve vulnerability page

### PR DESCRIPTION
**Pull request by reigj1**
Added CVE vulnerabilities page.

### CODE REVIEW

File: src/pages/security-advisories.tsx [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            <table className="w-full border border-black border-collapse">
+              <thead>
+                <tr>
+                  {[
+                    "CVE",
+                    "Brief description",
+                    "Reference",
+                    "Affected products",
+                    "Affected versions",
+                    "CVSS score",
```
Comment (venkkatesh-sekar) [inline src/pages/security-advisories.tsx:74]:
```suggestion
                    "CVSS 3.1",
```

File: src/pages/security-advisories.tsx [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+                    "Issued one",
```
Comment (venkkatesh-sekar) [inline src/pages/security-advisories.tsx:75]:
```suggestion
                    "Issued on",
```

File: src/pages/security-advisories.tsx [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    versions: ">= 0.20.0-beta.0, < 1.0.1",
+    cvss: "Critical (9.1/10)",
+    issued: "Feb 21, 2024",
+  },
+  {
+    cve: "CVE-2024-4435",
+    description:
+      "Stable BTreeMap memory leak when deallocating nodes with overflows",
+    reference:
+      "https://github.com/dfinity/stable-structures/security/advisories/GHSA-3rcq-39xp-7xjp",
```
Comment (venkkatesh-sekar) [inline src/pages/security-advisories.tsx:33]:
```suggestion
      "https://github.com/advisories/GHSA-3rcq-39xp-7xjp",
```

File: src/pages/security-advisories.tsx [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    versions: ">= 0.9.0, < 0.9.10",
+    cvss: "High (7.5/10)",
+    issued: "Dec 8, 2023",
+  },
+  {
+    cve: "CVE-2024-1631",
+    description:
+      "agent-js: Insecure Key Generation in `Ed25519KeyIdentity.generate`",
+    reference: "https://github.com/advisories/GHSA-c9vv-fhgv-cjc3",
+    products: "Agent-js @dfinity/auth-client (npm) @dfinity/identity (npm)",
```
Comment (venkkatesh-sekar) [inline src/pages/security-advisories.tsx:23]:
```suggestion
    products: "@dfinity/auth-client (npm) @dfinity/identity (npm)",
```

**Comment by venkkatesh-sekar** [comment]
Thanks for working on this @reigj1. Left some comments for some typos I had in the data.
Could we also add @dfinity/product-security as the owner for this page. 

Otherwise LGTM!

**Comment by venkkatesh-sekar** [comment]
Thanks @reigj1, LGTM now!

**Review by DeanDfinity — APPROVED** [review]

**Pull request merged**